components/dashboard/src/App.tsx

```diff
@@ -397,6 +398,9 @@ function App() {
                             path={projectsPathMainWithParams}
                             render={(props) => {
                                 const { resourceOrPrebuild } = props.match.params;
+                                if (resourceOrPrebuild === "events") {
```

Comment:
Sths strange here:
 1. With this code I would have expected to find this view at `/t/myteam/myproject/events`, but when I do so it drops into "PrebuildDetails" (fallback below).
 2. When looking at the code below, it seems we have a global events view at `/events`. I wonder what this is for - the user's events? :thinking: But if I visit that it says it cannot find it and redirects (to gitpod.io).
 3. In the description it says `/projects/myproject/events`: This is where I see an empty list with no data. It seems it's missing the "team" part, because the UI renders weird (see below), and we seem to [require it in code](https://github.com/gitpod-io/gitpod/pull/11405/files#diff-c41e3988d6100a5fc475d119437bc25b92fd009b1c68208148f9f494fbb795e3R61). Plus I didn't even knew we had that prefix - and wonder if we should at that view there :-D 
![image](https://user-images.githubusercontent.com/32448529/179217555-4182c523-b346-4a51-a4cc-78b79c723126.png)

Reply:
there is indeed a problem with `/t/team/project/events` location: 
https://at-prebuild-events.preview.gitpod-dev.com/t/team1/plain/events ✖️ 
https://at-prebuild-events.preview.gitpod-dev.com/projects/foobar/events ✔️ 

on 3.) that looks good. no team is required, the code reads ...

```
if (!teams || !projectSlug) {
            return;
        }
```
there is a project slug.
please be aware that the main menu is untouched for the 🛹

Reply:
fixed in https://github.com/gitpod-io/gitpod/compare/9031b74794fb79a4cf834f6f5cdb9ce4e3e5c8a6..262f9587439ee3350c5573c2889d1be54164bb63